packages/fluentui/react-builder/src/components/SRMC-Definitions.tsx

```diff
@@ -0,0 +1,222 @@
+        'aria-required=true': 'Required',
+      }, // End input:checkbox
+      'role=checkbox': {
+        'aria-checked=true': 'checked',
+        'aria-checked=false': 'not checked',
+        'aria-checked=mixed': 'partially checked',
+        'aria-required=true': 'Required',
+      }, // End role=checkbox
+      'input:radio': {
+        'checked=true': 'checked',
+        'checked=false': 'not checked',
+        'aria-required=true': 'Required',
+      }, // End input:radio
+      'role=radio': {
+        'aria-checked=true': 'checked',
+        'aria-checked=false': 'not checked',
+        'aria-required=true': 'Required',
+      }, // End role=radio
+      'role=combobox': {
+        'aria-invalid=true': 'invalid entry',
+        'aria-required=true': 'Required',
+      }, // End role=combobox
+      textarea: {
+        'aria-invalid=true': 'invalid entry',
+        'aria-required=true': 'Required',
+        '[extra1]': 'Contains text',
+      }, // End textarea
+      a: {
+        'aria-expanded=true': 'expanded',
+        'aria-expanded=false': 'collapsed',
+        'aria-haspopup=true': 'Has Popup menu',
+        'aria-haspopup=menu': 'Has Popup menu',
+        'aria-haspopup=listbox': 'Has Popup listbox',
+        'aria-haspopup=tree': 'Has Popup tree',
+        'aria-haspopup=grid': 'Has Popup grid',
+        'aria-haspopup=dialog': 'Has Popup dialog',
+        'aria-haspopup=': '',
+      }, // End a
+      'role=menuitem': {
+        'aria-haspopup=true': 'submenu',
+        'aria-haspopup=menu': 'submenu',
+        'aria-haspopup=listbox': 'submenu',
+        'aria-haspopup=tree': 'submenu',
+        'aria-haspopup=grid': 'submenu',
+        'aria-haspopup=dialog': 'submenu',
+        'aria-haspopup=': '',
+      }, // End role=menuitem
+      'role=menuitemcheckbox': {
+        'aria-checked=true': 'checked',
+        'aria-checked=false': 'not checked',
+      }, // End role=menuitemcheckbox
+      'role=menuitemradio': {
+        'aria-checked=true': 'checked',
+        'aria-checked=false': 'not checked',
+      }, // End role=menuitemradio
+      select: {
+        'aria-invalid=true': 'invalid entry',
+        'aria-required=true': 'Required',
+      }, // End select
+      'role=switch': {
+        'aria-checked=true': 'Pressed On',
+        'aria-checked=false': 'Off',
+      }, // End role=switch
+      'role=tab': {
+        'aria-selected=true': 'Selected',
+        'aria-selected=false': '',
+      }, // End role=tab
+      'role=option': {}, // End role=option
+    }, // End Win/JAWS
+  }, // End stateStrings
+
+  // Element usage strings for each platform and definition name
+  usageStrings: {
+    'Win/JAWS': {
+      button: {
+        '[default]': 'To activate press Enter.',
+        'aria-pressed=true': 'To toggle the state press spacebar.',
+        'aria-pressed=false': 'To toggle the state press spacebar.',
+      }, // End button
+      'input:text': {
+        '[default]': 'Type in text.',
```

Comment:
I'm confused as to why the square brackets are needed

Reply:
The brackets are not needed. It is just to make it apparent that it is not a state attribute or element definition. it from the state attributes or element definitions which don't have brakets.

Reply:
Sorry for the confusing comment. It should have been like this:

The brackets are not needed. It is just to make it more apparent that it is not a specific state attribute like "checked=true".

Reply:
The brackets are not needed. It is just to make it more apparent that this definition is not a specific state attribute like "checked=true".